### PR TITLE
get organization + refresh token implementation

### PR DESCRIPTION
**Pull request by MashB**

### CODE REVIEW

File: user-management-spec.json [hunk 1]
```diff
@@ -110,6 +110,53 @@
...
+                            }
+                        }
+                    },
+                    "500": {
+                        "description": "Internal server error."
+                    }
+                },
+                "security": [
+                    {
+                        "AuthorizationToken": []
```
Comment (susrisha) [inline user-management-spec.json:155]: Does refresh token also need security authorization?

File: user-management-spec.json [hunk 2]
```diff
@@ -110,6 +110,53 @@
...
+                "responses": {
+                    "400": {
+                        "description": "Invalid request."
+                    },
+                    "200": {
+                        "description": "Successful response - Returns the access token object.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "array",
```
Comment (susrisha) [inline user-management-spec.json:141]: Is the schema correct here? can you post an example of the response?

**Review by susrisha — CHANGES_REQUESTED** [review]

File: user-management-spec.json (hunk 1 above)
Comment (MashB) [reply user-management-spec.json:155]: not required it does not check in code. I will update the spec

File: user-management-spec.json (hunk 2 above)
Comment (MashB) [reply user-management-spec.json:141]: schema is correct. I removed array wrapper as correction.

**Review by rdevalap — APPROVED** [review]
Had a look together with Naresh. Saw that his comments are also integrated.
Approving the PR

**Review by susrisha — APPROVED** [review]
Approved

**Pull request merged**